react/tic-tac-toe/src/app/screens/App/components/Results/components/CustomRow/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class CustomRow extends Component {
+  renderItem = item => {
```

Comment:
If you're not having any other logic within this method, you can just return it using the default arrow function behavior.